app/models/batch_context.rb

```diff
@@ -23,7 +23,8 @@ class BatchContext < ApplicationRecord
     'file' => 3,
     'media' => 4,
     '3d' => 5,
-    'document' => 6
+    'document' => 6,
+    'maps' => 7
```

Comment:
Note, it has to be `maps` because `map` is a reserved keyword for enums:

```You tried to define an enum named "content_structure" on the model "BatchContext", but this will generate a class method "map", which is already defined by ActiveRecord::Relation.```